Util/remove_singleton.m

```diff
@@ -0,0 +1,36 @@
+function [nssize] = remove_singleton(sizearray)
```

Comment:
I agree with this one, should be removeSingleton, but not with the iscell,and friends because it's a moniker matlab uses a lot (for example `isunix,`isstr`, etc).